testing/py-bandit/APKBUILD

```diff
@@ -0,0 +1,47 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+pkgname=py-bandit
+_pkgname=${pkgname#*-}
+pkgver=1.4.0
+pkgrel=0
+pkgdesc="Security oriented static analyser for python code"
+url="https://wiki.openstack.org/wiki/Security/Projects/Bandit"
+arch="noarch"
+license="Apache2"
+depends="python"
+makedepends="python-dev python3-dev py-setuptools"
+options="!check"
```

Comment:
Could you add a comment why there is no check function, e.g. because of the lack of upstream test suite? Is there a simple quick smoke test you could add (e.g. invocation of the resulting binary/code) to make sure that at least something didn't go terribly wrong?